pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[tool.pytest.ini_options]
+pythonpath = [
+  "."
```

Comment:
we need to add `.` during testing so it can find the nlp4all module

Reply:
I don't get this

Reply:
Well when you run something in Python, by default it will only look for modules (`import x`) in paths specified in the python path variable. When you want to make it look in the directory you're working with you need to specify `.` (the current directory) so we can say `import nlp4all`

Reply:
Got it